packages/webdriver-utils/src/providers/automateProvider.js

```diff
@@ -1,11 +1,127 @@
+import utils from '@percy/sdk-utils';
```

Comment:
lets import individual functions here which is required.

```
import { TimeIt } from '@percy/sdk-utils';
```

Reply:
This needs change in `package.json`, we can pick this as in separate task to reconcile this.

Reply:
try removing `type: "module"`  does it help or is it a bigger change?

Reply:
There's not type mentioned in `sdk-utils` and adding there works for the webdriver utils but the sdk-util import statement will start breaking.